terraform-provider-google.yaml

```diff
@@ -16,7 +16,7 @@ pipeline:
     with:
       repository: https://github.com/hashicorp/terraform-provider-google
       tag: v${{package.version}}
-      expected-commit: 230df122485d58273235158c6bcd246f39f78b41
+      expected-commit: 2f104745123b975ba96b1eb9db0deab4caa25a5b
```

Comment:
Same here, you'll want to update `version`, `epoch`, and `expected-commit` to match

https://github.com/wolfi-dev/os/blob/main/terraform-provider-google.yaml#L1-L19